src/charactersheet/viewmodels/dm/initiative-tracker/index.html

```diff
@@ -35,22 +35,43 @@
       <table class="table table-responsive table-ac-bordered table-hover">
         <thead class="ac-table-header">
           <tr>
-            <th class="col-xs-7 col-md-8 col-lg-9">
-              <!-- ko if: state() === State.called -->
-              Participants (unsorted)
-              <!-- /ko -->
-              <!-- ko if: state() === State.started -->
-              Initiative Order
-              <!-- /ko -->
-            </th>
-            <th class="col-xs-4 col-md-3 col-lg-2 text-center">
-              <button
-                class="btn btn-xs btn-info"
-                data-bind="click: rollForInitiative, enable: canReRoll"
-                type="button"
-              >
-                Roll for Initiative
-              </button>
+            <th class="col-xs-11" colspan="2">
+              <div class="d-flex" style="width: 100%; justify-content: space-between;">
+                <div>
+                  <!-- ko if: state() === State.called -->
+                  Participants (unsorted)
+                  <!-- /ko -->
+                  <!-- ko if: state() === State.started -->
+                  Initiative Order
+                  <!-- /ko -->
+                </div>
+                <div class="text-right">
+                  <button
+                    class="btn btn-xs btn-default mr-1"
+                    data-bind="
+                      click: toggleIncludeBonuses,
+                      state() === State.called,
+                    "
+                    type="button"
+                  >
+                    <!-- ko if: includeBonuses -->
+                    <i class="fa fa-times-circle"></i>
+                    Exclude Bonuses
+                    <!-- /ko -->
+                    <!-- ko ifnot: includeBonuses -->
+                    <i class="fa fa-plus-circle"></i>
+                    Include Bonuses
```

Comment:
do you think this is descriptive enough given the context of the initiative tracker? do you think it would help to add a subtext describing what is meant/expected by these buttons?

Reply:
I'm open to change the button text but I would resist adding more text below it. The functionality is no -destructive so having people just experiment and see what it does won't hurt. The bonuses going in and out makes it clear imo.

Reply:
ooooh, that clears things up. thanks 👍